### DISCUSSION TITLE
doubt

### DISCUSSION THREAD

**Discussion by vamshipotharaveni** [codewars/codewars.com#3089] [General]
hello

**Comment by vamshipotharaveni** [comment]
am provided code is right but it showing to me as failled as a test case whats wrong with my code

**Reply by hobovsky** [reply]
See the [Troubleshooting guide](https://docs.codewars.com/training/troubleshooting).

**Comment by hobovsky** [comment]
Github discussions are not meant for help with individual kata. Please write a post labeled as `question` in kata discourse, or come for help to `#help-solve` channel of Codewars Discord.

I am going to close this discussion as off-topic.

**Reply by vamshipotharaveni** [reply]
ok
